test/index.js

```diff
@@ -9,6 +9,7 @@ var grunt = require('grunt'),
 test('Grunt-localizr', function (t) {
```

Comment:
We _could_ create a separate test file `test/localeList.js` which isolates localeList. I've done that in my local repo to work through my code review. It could help us to more easily test any other corner case assumptions.